### DISCUSSION TITLE
NSFW content depicting actual people is not permitted message

### DISCUSSION THREAD

**Discussion by silvern1n** [civitai/civitai#320] [General]
I keep receiving the NSFW content depicting actual people is not permitted warning when trying to review LoRAs and textual inversions although my images do not contain any NSFW content. I am unsure what about the images is triggering it, but it has been a reoccurring issue.

![civitai](https://user-images.githubusercontent.com/126818119/222528378-358fc187-a9de-4b89-8ba2-3895487ec187.jpg)

**Comment by WASasquatch** [comment]
Put some better clothes on that girl. 

But in all reality I'm sure the detector has a threshold of cleavage-to-boom ratio before it's NSFW. I'd probably flag a couple of those as NSFW on forums I mod to be blurred. 

NSFW is more about what's appropriate in work settings and such, not if it's [explicitly] "nude".

**Reply by JohnMadden123456** [reply]
We really need someone to make Burka LORA to make everything look 100% halal in previews
